### DISCUSSION TITLE
Seems like a "Save mission on all players disconnect" is not working for us

### DISCUSSION THREAD

**Discussion by logimy** [Vdauphin/HeartsAndMinds#1597] [Question and answer]
Title.
I'm not sure if it's a scenario bug or our server setting, but I have no clue what can cause it. We have resistant turned on in server settings and 3 headless clients loaded. 
![image](https://github.com/Vdauphin/HeartsAndMinds/assets/12511432/9596902e-7578-458b-a9fd-2d336f53e348)

The parameter is "On" but we found out that it's not triggering for some reason. Regular save working fine

Any help would be much appreciated.

**Comment by Vdauphin** [comment]
Hello,

Just tested and it works as intended with headless and even when connection is lost
Hard to tell from here, may be enable save on restart

Cheers
